### DISCUSSION TITLE
Where i can find the config file in mac? (Homebrew installation)

### DISCUSSION THREAD

**Discussion by Raichard** [streamlink/streamlink#3557] [Q&A] [answered]
In which subfolder do I have to search for that file? I tried specifying the streamlink to use another file I made but throw out the explanation to use -h for help.

**Solution:**
Create subfolders in the `.config` folder of the home directory (in short create subfolders in the [macOS home directory](https://www.cnet.com/how-to/how-to-find-your-macs-home-folder-and-add-it-to-finder/) like this: `~/.config/streamlink/config`) and the `streamlinkrc` rename it to `config` to match the path before mentioned

Some folders are hidden, use the option "go"->"go to folder" and write the name of the folder (in this case: `.config` when you are in the home directory)

**Comment by gravyboat** [comment, marked as answer]
We have some documentation on this [here](https://streamlink.github.io/cli.html#configuration-file). On Mac (Unix-like (POSIX)) the configuration file won't be created automatically, you'll have to do it.

**Reply by Raichard** [reply]
I have tried to do it with that command, but it does not work, after using the command I try to use the program and it remains the same as if it did not read the configuration:

I just get this:

> MacBook-Air-de-Ricardo:~ richi$ streamlink --config /Users/richi/Documents/programas/streamlinkrc
usage: streamlink [OPTIONS] <URL> [STREAM]
Use -h/--help to see the available options or read the manual at https://streamlink.github.io
MacBook-Air-de-Ricardo:~ richi$

**Reply by Raichard** [reply]
thanks by the way

**Reply by Raichard** [reply]
I managed to get it to work, I don't know if in the right way, I made the stream launch together with the --config command and now it works, than you <3

**Reply by beardypig** [reply]
You can create `.streamlinkrc` in your home directory and it should read that automatically.

**Reply by Raichard** [reply]
> You can create `.streamlinkrc` in your home directory and it should read that automatically.

I don't understand what the "home directory" is, is it the directory where streamlink is installed?

**Reply by bastimeyer** [reply]
https://en.wikipedia.org/wiki/Home_directory

**Reply by Raichard** [reply]
Thanks, I was able to do that by creating subfolders in the `.config` folder of the home directory (in short create subfolders in the [macOS home directory](https://www.cnet.com/how-to/how-to-find-your-macs-home-folder-and-add-it-to-finder/) like this: `~/.config/streamlink/config`) and the `streamlinkrc` rename it to `config` to match the path before mentioned
